index.coffee

```diff
@@ -81,10 +82,10 @@ createSprockets = (config) ->
   paths = []
   for path in writeFiles(config.sprocketsBundles, sprockets, tmpPath)
     paths.push
-      included: true
-      served: true
+      included: path.included
+      served: path.served
       watched: config.autoWatch
-      pattern: path
+      pattern: path.pattern
```

Comment:
`path` now is an object that we can push directly onto paths.

`paths.concat writeFile(config.sprocketsBundles, sprockets, tmpPath)`